setup.py

```diff
@@ -129,7 +129,7 @@ def get_git_sha():
         "elasticsearch": ["elasticsearch-dbapi>=0.2.0, <0.3.0"],
         "exasol": ["sqlalchemy-exasol>=2.1.0, <2.2"],
         "excel": ["xlrd>=1.2.0, <1.3"],
-        "gsheets": ["gsheetsdb>=0.1.9"],
+        "gsheets": ["shillelagh>=0.2"],
```

Comment:
I would add `, <0.3` here, just in case.

Reply:
+1